tests/api_tests.py

```diff
@@ -141,6 +141,16 @@ def test_get_all_cameras(self):
         result = self.app.get(route)
         self.assertEquals(result.status_code, 200)
 
+    def test_login(self):
+        route = "/users/login"
+        result = self.app.get(route)
+        self.assertEquals(result.status_code, 200)
+
+    def test_(self):
+        route = "/users"
+        result = self.app.get(route)
+        self.assertEquals(result.status_code, 200)
+
```

Comment:
because the routes are post you gotta set it up like this:

```python
def test_add_new_camera(self):
        data = {"ip": "192.168.1.107", "nickname": "home-camera"}
        result = self.app.post(
            "/camera", data=json.dumps(data), content_type="application/json")
        self.assertEqual(result.status_code, 200)
```